tests/zipkin_test.py

```diff
@@ -733,7 +736,32 @@ def test_adding_sa_binary_annotation_without_sampling():
             service_name='test_service',
             host='1.2.3.4',
         )
-        assert context.logging_context is None
```

Comment:
Why are we removing this assertion?

Reply:
Because it's testing something else than what the test name says. Whether `logging_context` is independent of `add_sa_binary_annotation`, so asserting it here doesn't give you anything.